README.md

```diff
@@ -1 +1,3 @@
 # first-repo
+
+by Fara and Oli
```

Comment:
Can you change this to use my full name please? Thank you

Reply:
Hey Fara, no problem will add now.